_posts/adam/2022-06-24-five-positive-for-every-negative.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "Five Positive Interactions for Every Negative One"
+permalink: /@adam/five-positive-for-every-negative
+image: assets/img/color-wars.jpeg
+description: One piece of leadership advice that has stuck with me is the idea that for every negative interaction you have with someone, you need to have 5 positive ones to offset it
+tags: tails management
+---
+
+<div class="text-center">
+  <img src="/assets/img/color-wars.jpeg"/>
+  <figcaption style="margin-top: 0px">That's me in the blue shark hat</figcaption>
+</div>
+
+## Residence Advising
+
+When I was a [Residence Advisor back at UBC](https://vancouver.housing.ubc.ca/jobs/), I received all kinds of coaching and advice about how to be a better leader. One piece of information has stuck with me ever since and it is the idea that for every negative interaction you have with someone, you need to have 5 positive ones to offset the negative one. 
+
+As a residence advisor, it was easy to find yourself in the middle of a negative interaction.  Often they were minor, "hey, don't smoke so close to the building" or "you're being too loud, please be quiet".  Occasionally they were major, requiring a proper "Incident Report" to be written up.  Regardless of the scale it was immediately noticeable that after the negative interaction, tension between you and the other person involved grew.
+
+Even after just a few interactions, relations between you and an individual can quickly sour.  After a few more, you'll find yourself in the territory of no return where this person instantly assumes you're only going to say something negative to them and in return they treat you in an unfriendly manner. Which then subconsciously irritates you more causing you to look for something else they're doing wrong.
+
+Whether you're living on residence for a full school year or working in an office environment, conflict of some form is inevitable. You will still see or work with this person for a year or more. Let's take a deeper look at how this advice can help improve teams in a software engineering environment.
```

Comment:
year or more. Let's take a (double spacing in use so far)
look at how the 1:5 relationship advice can help